setup.py

```diff
@@ -2,9 +2,16 @@
 
 setup(
     name="city_metrix",
-    version="0.1.0",
+    version="0.1.1",
     description="Module to calculate various metrics on cities.",
-    packages=["city_metrix", "city_metrix.layers", "city_metrix.metrics"],
+    packages=["city_metrix", "city_metrix.layers", "city_metrix.metrics", "city_metrix.models", "city_metrix.models.building_classifier"],
```

Comment:
do you need to explicitly include `city_metrix.models.building_classifier`. or can you just include `city_metrix.models`?

Reply:
I need to use the `BuildingClassifier()` class in the `models/building_classifier/building_classifier.py`. In a package structure, does it only recognize subfolders as packages? 
I updated to use `find_packages()` now, so we don't need to list all the subpackages. That works?